### PR TITLE
My Portfolio Pull Request

### PR DESCRIPTION
**Pull request by mar4444**
What I did to my project:

- Created index.html file.
- Created style.css file.
- Used GitHub Flow
- Created desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,762 @@
...
+    display: none;
+  }
+
+  .menu-sec {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    margin-right: 71px;
+    line-height: 50px;
+  }
```
Comment (zieeco) [inline style.css]: - Hey, let's kindly increase the spaces between each items and increase the space towards the end as seen in the images below to improve our design:




*Figma design*
image


*your design*
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px 0 45px 0;
+  border-radius: 60px 0 0 0;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  margin-top: 114px;
+}
```
Comment (zieeco) [inline style.css]: - Hey, let's also use the correct background image for the desktop version of the contact form to make your design more consistent following the original design as much as you can. see the images below:


*Your design*
image



*Figma design*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mar4444,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice using media query.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions


- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @mar4444 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**